workflow/rules/common.smk

```diff
@@ -180,7 +180,9 @@ def aggregate_maltextract(wildcards):
 def _aggregate_utils(fmt, wildcards):
     """Collect common output for all aggregate functions. Returns a tuple
     of lists sample, taxid, and refid"""
```

Comment:
Are there still cases where this generates a refid?

Reply:
The aggregate_* functions are still needed, since we are still generating taxids that are unknown before execution time.

One thing I could do is to remove from `_aggregate_utils()` anything related to ref IDs. I am unsure whether this is the best idea, since in some cases we might want to avoid processing taxids where the ref ID is missing (as described in issue #56 )

What do you think?

Reply:
Sorry for the confusion, I was unclear. I was mainly referring to the docstring; refid isn't returned anymore, or?

Reply:
But to your comment, we probably still need to check for ref IDs in the function.

Reply:
It is returned right now. But yes, it's not needed so I will remove that